src/main/java/seedu/address/logic/commands/AddExamCommand.java

```diff
@@ -38,7 +40,7 @@ public class AddExamCommand extends Command {
             + PREFIX_TIME + "09:00";
 
     public static final String MESSAGE_SUCCESS = "Added exam to person with ID: %1$s";
-
+    private static final Logger logger = Logger.getLogger(AddExamCommand.class.getName());
```

Comment:
The other Logger class just writes the class name and don't have a get name method. maybe wanna change it to follow their format ?

![image](https://github.com/AY2324S2-CS2103T-F10-2/tp/assets/122339963/99c6aadc-f961-4d6e-a521-c27c8f522bc1)